config/ConfigExample/RemoteConfigViewController.swift

```diff
@@ -70,6 +70,20 @@ class RemoteConfigViewController: UIViewController {
     let settings = RemoteConfigSettings()
     settings.minimumFetchInterval = 0
     remoteConfig.configSettings = settings
+
+    // [START add_config_update_listener]
+    remoteConfig.addOnConfigUpdateListener { configUpdate, error in
+      guard error == nil else { return self.displayError(error) }
```

Comment:
For the else lock of the error check can we do 
`DispatchQueue.main.async {
       self.displayError(error)
}
return
`

Reply:
Either seems fine - I believe the issue is that this completion may not be called on the main thread and any UIAlertController work needs to happen on the main thread

Reply:
encapsulated the present in main async